src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -138,60 +149,107 @@ const TableHeader: React.FC<TableHeaderProps> = ({
     };
   }, [leaveSeat]);
 
+  const updateTableTitle = useCallback(
+    async (newTableTitle) => {
+      if (!user) return;
+
+      const venueTables = tables.map((table) => {
+        if (table.reference === tableOfUser?.reference) {
+          return { ...table, title: newTableTitle };
+        }
+        return table;
+      });
+
+      // Ideally we want to do this only when config.tables doesn't exist. Otherwise we want to update only a single table.
+      await updateVenue_v2(
+        { name: venueName, tables: venueTables },
+        user
+      ).catch(() => {
+        // @debt Replace generic error when proper error handling is added
+        setError(GENERIC_ERROR);
```

Comment:
Does the current backend function not return any `HttpsError`s at the moment?

Reply:
It does, but I have no way to simulate the errors. Maybe I can handle status code 500 and that's it.

Reply:
Removed generic error and added bugsnag to api in: https://github.com/sparkletown/sparkle/pull/1377/commits/de659957079dd74136f26d29e9b952be12add510

Reply:
Why removed the generic error?

Bugsnag is good for us, but it does nothing to communicate the error to the user. That's what the generic error part of this was doing. We want both.

Reply:
I'm displaying the server side error now. Not sure if this is what we prefer. But the `useAsyncFn` returns `.error.message` and this is being displayed atm.

Reply:
Ah, cool, that works then.

(once it's fixed to re-`throw` the error)